docs/toolhive/guides-vmcp/authentication.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Authentication
+description: Configure client and backend authentication for Virtual MCP Server.
+---
+
+Virtual MCP implements a two-boundary authentication model that separates client
+authentication from backend authentication, giving you centralized control over
+access while supporting diverse backend requirements.
+
+## Two-boundary authentication model
+
+```mermaid
+flowchart LR
+    subgraph Boundary1[" "]
+        direction TB
+        Client[MCP Client]
+        B1Label["**Boundary 1**<br>Client → Virtual MCP"]
+    end
+
+    subgraph vMCP[Virtual MCP Server]
+        Auth[Token Validation]
+        Exchange[Token Exchange]
+    end
+
+    subgraph Boundary2[" "]
+        direction TB
+        B2Label["**Boundary 2**<br>Virtual MCP → Backend APIs"]
+        GitHub[GitHub API]
+        Jira[Jira API]
+    end
+
+    Client -->|"Token: aud=vmcp"| Auth
+    Auth --> Exchange
+    Exchange -->|"Token: aud=github-api"| GitHub
+    Exchange -->|"Token: aud=jira-api"| Jira
+```
+
+**Boundary 1 (Incoming):** Clients authenticate to Virtual MCP using a token
+with `aud=vmcp`. This is your organization's identity layer.
```

Comment:
Let's be more generic here and say something along the lines of "vMCP-scoped token with the appropriate audience and scopes".

Reply:
Fixed in 9def7cc - updated wording to be more generic about token acquisition